extensions/iam/daps/src/test/java/org/eclipse/dataspaceconnector/iam/daps/DapsIntegrationTest.java

```diff
@@ -37,8 +36,8 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
-@IntegrationTest
 @ExtendWith(EdcExtension.class)
+@org.eclipse.dataspaceconnector.iam.daps.annotations.DapsIntegrationTest
```

Comment:
import

Reply:
Tag and class name is same that's why FQN used. Will see again if tag naming can be changed.

Reply:
Renamed and Updated. thanks.